django_short_urls/auth.py

```diff
@@ -11,6 +11,20 @@
 from django.http import HttpResponse
 
 
+def _safe_base64_decode(encoded):
+    """Method that base64 decodes a string, while gracefully handing insufficient padding.
```

Comment:
Hmmm… it's kind of an indirect reference, but probably worth linking in the code comments, at least for background